client/src/components/SlotDialog.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <section class="bg-light dialog-bg-image">
```

Comment:
Was passiert hier?

Reply:
Da wir das mehrfach verwendet haben, habe ich einen Komponente gebaut die andere wrappen kann.
in der mitte ist ein slot.

siehe user register wie man den dann verwendet